idea/idea-fir/src/org/jetbrains/kotlin/idea/quickfix/MainKtQuickFixRegistrar.kt

```diff
@@ -26,6 +27,22 @@ class MainKtQuickFixRegistrar : KtQuickFixRegistrar() {
                 isRedundant = true
             )
         )
+
+        val addAbstractModifierFactory = AddModifierFix.createFactory(KtTokens.ABSTRACT_KEYWORD)
+        registerPsiQuickFixes(KtFirDiagnostic.NonAbstractFunctionWithNoBody::class, AddFunctionBodyFix, addAbstractModifierFactory)
+
+        val addAbstractToClassFactory = AddModifierFix.createFactory(KtTokens.ABSTRACT_KEYWORD, KtClassOrObject::class.java)
+        val removeAbstractModifierFactory = RemoveModifierFix.createRemoveModifierFromListOwnerPsiBasedFactory(KtTokens.ABSTRACT_KEYWORD)
```

Comment:
Let's not mix up factories creation and registration. E.g, factories can be moved to `AddModifierFix`/`RemoveModifierFix` companions